todo_app/app.py

```diff
@@ -1,36 +1,66 @@
+import json
 import os
+
+from flask.wrappers import Response
+
+from todo_app.items.item import ItemEncoder
+from todo_app.view_model import ViewModel
 from todo_app.items.trello.transport import TrelloTransport
 from flask import Flask, render_template, request
 
-from todo_app.flask_config import Config
-
 from todo_app.items.trello.items import Items
 
-app = Flask(__name__)
-app.config.from_object(Config)
-trello_transport = TrelloTransport(os.environ.get('TRELLO_API_KEY'), 
+def create_app():
+    app = Flask(__name__)
+    # We specify the full path and remove the import for this config so
+    # it loads the env variables when the app is created, rather than when this file is imported
+    app.config.from_object('todo_app.flask_config.Config')
+
+    trello_transport = TrelloTransport(os.environ.get('TRELLO_API_KEY'), 
         os.environ.get('TRELLO_SERVER_TOKEN'))
-item_registry = Items(trello_transport, os.environ.get('TRELLO_BOARD_ID'))
+    item_registry = Items(trello_transport, os.environ.get('TRELLO_BOARD_ID'))
+
+    @app.route('/')
+    def index():
+        app.logger.info("Request received to load root")
+        return render_template('index.html')
+
+    @app.route('/todo')
+    def get_all_items():
+        items = sorted(item_registry.get_items(), key=lambda item: item.status, reverse=True)
+        item_view_model = ViewModel(items)
```

Comment:
Given that you're writing an SPA, you've tried to include a view model as the instructions ask for but it doesn't really fit.

A view model is part of the MVC pattern. It would make more sense for this route to just get the correct items in the first place.

Reply:
Yeah, it did feel awkward. It did make testing somewhat easier but yes will likely need to be modified going forward.